### PR TITLE
Libvpx 1.4.0

### PR DESCRIPTION
**Pull request by uyjulian**

### CODE REVIEW

File: Library/Formula/libvpx.rb [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 class Libvpx < Formula
   homepage 'http://www.webmproject.org/code/'
-  url 'https://webm.googlecode.com/files/libvpx-v1.3.0.tar.bz2'
-  sha1 '191b95817aede8c136cc3f3745fb1b8c50e6d5dc'
+  url "https://chromium.googlesource.com/webm/libvpx",
+    :using => :git,
+    :revision => "c74bf6d889992c3cabe017ec353ca85c323107cd"
+  version "1.4.0"
```
Comment (mikemcquaid) [inline Library/Formula/libvpx.rb]: Are they no longer publishing tarballs? Also, we need a tagged revision here.

Comment (uyjulian) [inline Library/Formula/libvpx.rb]: yeah, they aren't doing tarballs anymore

File: Library/Formula/libvpx.rb [hunk 2]
```diff
@@ -2,8 +2,10 @@
 
 class Libvpx < Formula
   homepage 'http://www.webmproject.org/code/'
-  url 'https://webm.googlecode.com/files/libvpx-v1.3.0.tar.bz2'
-  sha1 '191b95817aede8c136cc3f3745fb1b8c50e6d5dc'
+  url "https://chromium.googlesource.com/webm/libvpx",
```
Comment (mikemcquaid) [inline Library/Formula/libvpx.rb]: Use `https://chromium.googlesource.com/webm/libvpx.git` and you can drop the `:using` line.

File: Library/Formula/libvpx.rb [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 class Libvpx < Formula
   homepage 'http://www.webmproject.org/code/'
-  url 'https://webm.googlecode.com/files/libvpx-v1.3.0.tar.bz2'
-  sha1 '191b95817aede8c136cc3f3745fb1b8c50e6d5dc'
+  url "https://chromium.googlesource.com/webm/libvpx", 
+    :using => :git, 
+    :tag => "v1.4.0", 
+    :commit => "c74bf6d889992c3cabe017ec353ca85c323107cd"
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/libvpx.rb]: Need to remove the `revision` on version updates.

File: Library/Formula/libvpx.rb (hunk 1 above)
Comment (xu-cheng) [inline Library/Formula/libvpx.rb]: We can use the tarball from its official Github mirror: https://github.com/webmproject/libvpx/releases

Comment (mikemcquaid) [inline Library/Formula/libvpx.rb]: Good idea :+1:

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**